libs/blocks/merch/checkout-context.js

```diff
@@ -0,0 +1,30 @@
+const DEFAULT_WORKFLOW_STEP = 'email';
+const DEFAULT_CLIENT_ID = 'adobe_com';
```

Comment:
why have you actually created a separate file for the checkout context? All this can be moved to the merch.js block imo.

Reply:
good point, each file is loaded separately, better to keep the logic concise and in merch.js

Reply:
I find it easier to locate checkout logic in a dedicated file, but if it affects performance I am ok to move it back to merge.js.